libcodechecker/cmd/cmd_line_client.py

```diff
@@ -239,14 +239,23 @@ def add_filter_conditions(report_filter, filter_str):
     severity, checker, path = map(lambda x: x.strip(), filter_str.split(':'))
 
     if severity:
-        report_filter.severity = \
-            shared.ttypes.Severity._NAMES_TO_VALUES[severity.upper()]
+        report_filter.severity\
+            = shared.ttypes.Severity._NAMES_TO_VALUES[severity.upper()]
     if checker:
         report_filter.checkerId = '*' + checker + '*'
     if path:
         report_filter.filepath = path
 
 
+def add_server_arguments(parser):
```

Comment:
Because of the tests for suppress, either keep this, or copy the contents into it's call site at `suppress` arguments.